src/profiler.cpp

```diff
@@ -15,6 +15,7 @@
 #include "profiler.h"
 #include "perfEvents.h"
 #include "ctimer.h"
+#include <time.h>
```

Comment:
The last minor thing: do not interleave standard headers with project header files.
The recommended #include order is the following:
```
<standard c++ headers>
<standard c headers>
"currentModule.h"
"other headers"
```
all in alphabetical order.